static/src/javascripts/projects/commercial/modules/comscore.spec.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import config from 'lib/config';
+import { init, _ } from './comscore';
+
+const getComscoreScripTags = (): NodeList<HTMLElement> =>
+    document.querySelectorAll('script#comscore');
+
+const isValidComscoreScriptTag = (tag: HTMLElement) =>
+    tag instanceof HTMLScriptElement &&
+    tag.src === _.comscoreSrc &&
+    tag.async === true;
+
+describe('comscore init', () => {
+    beforeEach(() => {
+        config.set('switches.comscore', true);
+    });
+
+    it('should do nothing if the comscore switch is off', () => {
+        config.set('switches.comscore', false);
+        init();
+
+        const comscoreTags = getComscoreScripTags();
+        expect(comscoreTags.length).toBe(0);
+    });
+
+    it('should drop the corect script tag if the comscore switch is on', () => {
+        config.set('switches.comscore', true);
+        init();
+
+        const comscoreTags = getComscoreScripTags();
+        expect(comscoreTags.length).toBe(1);
+        expect(isValidComscoreScriptTag(comscoreTags[0])).toBe(true);
```

Comment:
This feels to me like one of the following are true:

* it is two separate tests
* the first expectation isn't adding much value

I'm not sure which, but as elsewhere best to try and keep to one assertion per test. If we're the first case then perhaps we need a separate test that says `"only one comscore script gets added"`?

Reply:
👍